internal/pkg/config/bgp_configs.go

```diff
@@ -1416,7 +1416,9 @@ type BmpServerConfig struct {
 	// gobgp:route-mirroring-enabled's original type is boolean.
 	// Enable feature for mirroring of received BGP messages
 	// mainly for debugging purpose.
-	RouteMirroringEnabled bool `mapstructure:"route-mirroring-enabled" json:"route-mirroring-enabled,omitempty"`
+	RouteMirroringEnabled bool   `mapstructure:"route-mirroring-enabled" json:"route-mirroring-enabled,omitempty"`
+	SysName               string `mapstructure:"SysName" json:"SysName,omitempty"`
+	SysDescr              string `mapstructure:"SysDescr" json:"SysDescr,omitempty"`
 }
```

Comment:
bgp_configs.go is automatically generated. you can modify it directly
https://github.com/osrg/gobgp/tree/master/tools/pyang_plugins

Reply:
let me know if you like me to take care of changes to bgp_configs.go.

Reply:
Thanks, I fixed it, and I also fixed the pyang plugins document. Is there have something error please help me fix it out.